conda-recipe/meta.yaml

```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
 #===============================================================================
 
-{% set version = '2021.1' %}
-{% set buildnumber = 4 %}
+{% set version = '2021.2' %}
```

Comment:
why not 2021.3 ? It's current one. More over assuming created branches master should be 2021.4